test/core/src/temalab/logger/ConsoleLogger.java

```diff
@@ -0,0 +1,24 @@
+package temalab.logger;
+
+/**
+ * Logs to the standard output.
+ * Additionally, it changes (and resets) the console's color based on the type of the log.
+ */
+public class ConsoleLogger implements Logger {
```

Comment:
A kódduplikáció csökkentése érdekében nagy valószínűséggel kelleni fog egy `StreamLogger` osztály is, ami tetszőleges `PrintSteam`/`OutputSteam`-ekre tud írni.

Reply:
Sőt lehet, hogy a `StreamLogger` lesz az egyetlen logger implementáció, és ha valaki saját helyre akar logolni, akkor csak készít egy `OutputStream`-et.